README.md

```diff
@@ -203,7 +203,7 @@ container's configuration. The `configFile` is a closure itself and requires you
 A [`FileCollection`](http://www.gradle.org/docs/current/javadoc/org/gradle/api/file/FileCollection.html) should be used as `files` attribute and `toDir` should be a `String`.
 Multiple configuration file destinations can be defined by creating more than one `configFile` closure.
 * `rmiPort`: The port to use when communicating with this server, for example to start and stop it.
-* `timeout`: The timeout (in ms) in which to determine if the container is successfully started or stopped (defaults to 120000ms).
+* `startupTimeout`: The timeout (in ms) in which to determine if the container is successfully started or stopped (defaults to 120000ms).
```

Comment:
Let's remove the word "container" here to be consistent. We can just use what the Cargo documentation says "The timeout after which Cargo reports an error if the container is not started or stopped".